vnext/PropertySheets/Application/Release.props

```diff
@@ -8,8 +8,9 @@
   </PropertyGroup>
 
   <ItemDefinitionGroup>
-    <ClCompile>
-    </ClCompile>
+    <Link>
+      <GenerateDebugInformation>DebugFull</GenerateDebugInformation>
```

Comment:
>DebugFull [](start = 32, length = 9)

Is the pattern to include "DebugFull" for ship?

Reply:
Yes.  This is just specifying how complete the PDB files are.  We still want full PDBs even on ship builds.